packages/mobile/src/screens/coin-redeem-screen/CoinRedeemScreen.tsx

```diff
@@ -0,0 +1,171 @@
+import { useCallback, useMemo } from 'react'
+
+import {
+  useArtistCoinByTicker,
+  useCoinRedeemAmount,
+  useCoinRedeemCodeAmount,
+  useRedeemCoin,
+  useRedeemCoinCode
+} from '@audius/common/api'
+import { route } from '@audius/common/utils'
+import { useRoute } from '@react-navigation/native'
+
+import {
+  Box,
+  Button,
+  Flex,
+  Paper,
+  Skeleton,
+  Text
+} from '@audius/harmony-native'
+import {
+  Screen,
+  ScreenContent,
+  ScrollView,
+  TokenIcon
+} from 'app/components/core'
+import { useNavigation } from 'app/hooks/useNavigation'
+import { useToast } from 'app/hooks/useToast'
+
+import { BannerSection } from '../coin-details-screen/components/CoinInfoCard'
+
+const messages = {
+  title: (ticker: string) => `Redeem $${ticker}`,
```

Comment:
sure the ticker doesn't already have $ on it? try with a prod coin, staging coins might be out of date